src/langcheck/metrics/eval_clients/_prometheus.py

```diff
@@ -51,32 +56,34 @@ def get_text_responses(
             A list of responses to the prompts. The responses can be None if the
             evaluation fails.
         """
-        response_texts = []
-        tqdm_description = tqdm_description or "Intermediate assessments (1/2)"  # NOQA: E501
-        for prompt in tqdm_wrapper(prompts, desc=tqdm_description):
```

Comment:
Two reasons:
- It's faster to process prompts and generate outputs at once instead of using a for loop. However, I couldn't find out a way to use tqdm without the for loop.
- There is an in-house progress bar in `vllm`. Using tqdm here will result in double bars.
```
Processed prompts: 100%|█████████████████████████████████████████████████████████| 3/3 [00:10<00:00,  3.58s/it, est. speed input: 70.57 toks/s, output: 40.69 toks/s]
```

Reply:
Ok got it! In that case, I think we can remove the `tqdm_description` param from `get_text_responses`, and also from `get_float_score`. For `get_float_score`, I think we can just not show a progress bar since it's just string finding